.NET/Microsoft.Recognizers.Text.DataDrivenTests/TestHelpers.cs

```diff
@@ -33,7 +33,7 @@ public static void InitFromTestContext(this TestResources resources, TestContext
             var className = context.FullyQualifiedTestClassName.Substring(classNameIndex + 1).Replace("Test", "");
             var recognizerLanguage = className.Split('_');
 
-            var directorySpecs = Path.Combine("..", "..", "..", "..", "Specs", recognizerLanguage[0], recognizerLanguage[1]);
+            var directorySpecs = Path.Combine("..", "..", "..", "..", "..", "Specs", recognizerLanguage[0], recognizerLanguage[1]);
```

Comment:
I'm not sure whether this extra level up in the directory tree was needed because of the csproj format update or because Visual Studio 2019 handles working paths differently.